generated/ridp/verify_identity_final_error_response.json

```diff
@@ -1,7 +1,7 @@
 {
   "verification_result": {
-    "response_code": "urn:openhbx:terms:v1:interactive_identity_verification#SUCCESS",
-    "response_text": "HS000000 Successful",
+    "response_code": "urn:openhbx:terms:v1:interactive_identity_verification#FAILURE",
+    "response_text": "Failed response - please see ref below.",
```

Comment:
Based on discussions with @benjaminrosenbaum on Slack, this is what we decided -

* Create separate directories (_each containing the `post` JSON files *and* the actual `response`_).
* Example: `/ridp_user_in_enroll/response.json` + `/ridp_user_in_enroll/post_verify_identity.json` + `/ridp_user_in_enroll/post_answer_questions.json`